fury/io.py

```diff
@@ -223,9 +230,12 @@ def save_image(arr, filename, compression_quality=75,
     if use_pillow:
         arr = np.flipud(arr)
         im = Image.fromarray(arr)
-        im.save(filename, quality=compression_quality)
+        im.save(filename, quality=compression_quality, dpi=dpi)
         return
 
+    if dpi != (72, 72):
```

Comment:
This will trigger the warning for any value different from `(72, 72)`, e.g. `(150, 150)`, which is not the intent of the warning. It should be `if not use_pillow:` (regardless of the DPI value), right?

Reply:
I saved images using vtk and it seems like it always saves them at 72 dpi. So I thought maybe I should put 72 as default for both the method but I do understand your point of view. I have updated the if else block.